RFCs/rfc-0010-zk-passport.md

```diff
@@ -0,0 +1,147 @@
+## RFP: ZKPassport
+
+### Abstract
+
+This proposal introduces a standard protocol for generating identity-related attestations from a Mina wallet. It aims to enhance the trust and integrity of digital identity systems by allowing users to prove the validity of their identity attributes without revealing the underlying data. This approach not only safeguards user privacy but also provides a robust framework for decentralized identity verification, making it applicable across various domains such as finance, healthcare, and online services. Through the implementation of ZKPassport, Mina wallet users can achieve higher levels of security and privacy in their digital interactions while ZKApps achieve a higher level of defense against sybil attacks and the ability to make their applications compliant with local laws.
+
+### How will users provide the correct documents?
+
+Most modern passports are Biometric passports or ePassports and contain a chip that can be scanned with NFC. These passports contain information regarding the passport such as country, age, date of birth, etc.
+
+When scanned via a trusted mobile app, users can confidently produce a ZK Proof attesting to specific data on their passport chip. To gain a better understanding of which data can be accessed, please read the [ICAO specification](https://www.icao.int/publications/pages/publication.aspx?docnum=9303). Additionally, the public keys for checking the validity of this data can be found [here](https://download.pkd.icao.int/).
+
+It's important to note that different issuer countries use different signature algorithms.
+
+### Specification
+
+Generating a proof will involve multiple steps:
+
+#### Register Circuit
+
+The register circuit is used for the following:
+
+- Verify the signature of the passport
+- Verify that the public key which signed the passport is part of the registry Merkle tree (a check of the Merkle roots will be performed on-chain)
```

Comment:
What is the registry merle tree in this case? I'm not sure there is one, unfortunately (or at least not a public one)